tests/memorial_test.cpp

```diff
@@ -202,6 +202,9 @@ TEST_CASE( "memorials" )
     check_memorial<event_type::opens_temple>(
         m, b, "Opened a strange temple." );
 
+    check_memorial<event_type::player_levels_spell>(
```

Comment:
Please look at what this function actually does. Hint: it creates an event of the given type, which in turn needs some arguments (that's what the assertion means). Compare this to the test for `learns_martial_art` above. (that one adds two more arguments: the character id and the style id). Note that the resulting string (which is checked by the function) will also be different (it contains the spell name), so you need to check for that as well.